strings-csharp/RandomAlphaNumericString/RandomAlphaNumericString/CryptographicallySecureMethods/SpanSecureMethod.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Security.Cryptography;
+
+namespace RandomAlphaNumericString
+{
+    public partial class Methods
+    {
+        public static string? SpanSecureMethod(int length)
+        {
+            Span<byte> data = stackalloc byte[4 * length];
+            using (RandomNumberGenerator rng = RandomNumberGenerator.Create())
+            {
+                rng.GetBytes(data);
+            }
```

Comment:
We can use the static methods on `RandomNumberGenerator` here without creating a new one instance